.pipelines/templates/template-run-rp.yml

```diff
@@ -0,0 +1,83 @@
+parameters:
+  workingDirectory: ""
+steps:
+  - script: |
```

Comment:
is "set -xe" implied? what happens is one command fails? does it just keep running?

Reply:
yep keep running for now. We can improve this for sure. I just want to make sure we are reaching out `az group delete` command at the end to clean as much as possible.